modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/PageMemoryMvStorageIoModule.java

```diff
@@ -28,7 +28,7 @@
 import org.apache.ignite.internal.storage.pagememory.mv.io.VersionChainMetaIo;
 
 /**
- * {@link PageIoModule} related to {@link PageMemoryMvPartitionStorage} implementation.
+ * {@link PageIoModule} related to {@link AbstractPageMemoryMvPartitionStorage} implementation.
```

Comment:
Looks weird when it points to an abstract class. I think we can list 2 MvTableStorage implementations here, would be better.

Reply:
fix it